apps/src/StudioApp.js

```diff
@@ -2980,6 +2980,7 @@ StudioApp.prototype.handleUsingBlockly_ = function (config) {
     typeHints: utils.valueOr(config.level.showTypeHints, false),
     isBlocklyRtl:
       getStore().getState().isRtl && config.levelGameName !== 'Jigsaw', // disable RTL for blockly on jigsaw
+    isJigsaw: config.levelGameName === 'Jigsaw',
```

Comment:
super nit/not attached: Could store this in a variable before the object is declared and use `isJigsaw` on the line above, too